internal/controller/storageclaim_controller.go

```diff
@@ -365,6 +365,9 @@ func (r *StorageClaimReconciler) reconcilePhases() (reconcile.Result, error) {
 					storageClass = r.getCephRBDStorageClass()
 				}
 				err = utils.CreateOrReplace(r.ctx, r.Client, storageClass, func() error {
+					if err := r.own(storageClass); err != nil {
```

Comment:
before the regression the expectation was createorreplace code creates the ownerereference and is there any other place that it isn't intended?